nmi_mysql/nmi_mysql.py

```diff
@@ -18,14 +18,13 @@
 
 class DB(object):
 
-    def __init__(self, conf, autoconnect=False):
+    def __init__(self, conf):
```

Comment:
I think we can create a deprecated function to still allow the autoconnect to be passed in and show warnings.

This would allow backward compatibility

Or, change the parameter to indicate create one connection instead of auto connect, which would be doing the same thing but optional.